tester.R

```diff
@@ -0,0 +1,99 @@
+library(tidyverse)
```

Comment:
This is great to see. I've always wanted to implement unit testing, and now I think we have the space and personnel to actually do it. More to say. I'll save it for slack.